plexupdate.sh

```diff
@@ -255,29 +259,29 @@ if [ "${AUTOUPDATE}" == "yes" ]; then
 fi
 
 # Sanity check
-if [ "${EMAIL}" == "" -o "${PASS}" == "" ] && [ "${PUBLIC}" == "no" ] && [ ! -f /tmp/kaka ]; then
-	echo "Error: Need username & password to download PlexPass version. Otherwise run with -p to download public version." >&2
+if [ -z "${EMAIL}" -o -z "${PASS}" ] && [ "${PUBLIC}" = "no" ] && [ ! -f /tmp/kaka ]; then
+	echo "ERROR: Need username & password to download PlexPass version. Otherwise run with -p to download public version." >&2
 	cronexit 1
 fi
 
-if [ "${AUTOINSTALL}" == "yes" -o "${AUTOSTART}" == "yes" ]; then
+if [ "${AUTOINSTALL}" = "yes" -o "${AUTOSTART}" = "yes" ]; then
 	id | grep -i 'uid=0(' 2>&1 >/dev/null
 	if [ $? -ne 0 ]; then
-		echo "Error: You need to be root to use autoinstall/autostart option." >&2
+		echo "ERROR: You need to be root to use autoinstall/autostart option." >&2
 		cronexit 1
 	fi
 fi
 
 
 # Remove any ~ or other oddness in the path we're given
-DOWNLOADDIR="$(eval cd ${DOWNLOADDIR// /\\ } ; if [ $? -eq 0 ]; then pwd; fi)"
```

Comment:
Going to investigate this one further. My command-line tests with `eval` removed were fine, but I want to check for any edge cases where it might break.